commands/roles.js

```diff
@@ -30,23 +70,25 @@ const addRole = async (interaction, guildSettings) => {
 };
 
 const removeRole = async (interaction, guildSettings) => {
-	const chosenRole = getChosenRole(interaction, guildSettings);
+	const roleProvided = interaction.options.getRole('role');
 
-	if (
-		!interaction.member.roles.cache.some(
-			(role) => role.name === chosenRole.name
-		)
-	) {
+	if (!isManagedRole(roleProvided)) {
```

Comment:
Should this be the inverse? If it's not a managed role we should go ahead and remove it.

Reply:
Yes it should, thank you I have fixed this now